bigbone/src/integrationTest/kotlin/social/bigbone/TestHelpers.kt

```diff
@@ -0,0 +1,64 @@
+package social.bigbone
+
+import social.bigbone.api.Scope
+import social.bigbone.api.entity.Application
+import social.bigbone.api.entity.MediaAttachment
+import social.bigbone.api.entity.Token
+import java.io.File
+
+/**
+ * This class provides helper methods used in integration tests which are not specific to any
+ * version of Mastodon. If a version-specific helper needs to be written, please create a new
+ * class (e.g. V402TestHelpers.kt) in the appropriate integration test directory.
+ */
+object TestHelpers {
+    /**
+     * Uploads an image file using the POST /api/v1/media REST call.
+     * @param filename the filename. Must be located in the "resources" directory".
+     * @param mediaType the media type of the file (e.g. "image/jpg", ...).
+     * @param client an instance of MastodonClient. This instance is used to conduct the upload.
+     * @return the media attachment information.
+     */
```

Comment:
How strict do we want to be about documentation for our test classes? This is the only method documentation in the class - but I guess the rest is pretty straightforward.

Reply:
Yes, well, my vote basically is for having the code written in a way that it does not need KDoc. And in this case, it is pretty obvous what the method does. So I can remove it. I wouldn't be strict with KDoc on test code.....